liberapay/elsewhere/_base.py

```diff
@@ -144,9 +144,10 @@ def api_get(self, domain, path, sess=None, **kw):
         return self.api_request('GET', domain, path, sess=sess, **kw)
 
     def api_error_handler(self, response, is_user_session, domain):
+        response_text = response.text  # for Sentry
```

Comment:
![screenshot_20180409_143620](https://user-images.githubusercontent.com/1581590/38498027-6103839e-3c03-11e8-9f60-eca25d1e1e70.png)

Reply:
I've added `response_text` to the "safe" list:

![screenshot_20180409_145122](https://user-images.githubusercontent.com/1581590/38498680-7c25e0de-3c05-11e8-8c22-49cafc22523d.png)